lib/vdsm/network/link/setup.py

```diff
@@ -20,38 +20,64 @@
 from __future__ import absolute_import
 from __future__ import division
 
-from vdsm.network.link.bridge import Bridge
```

Comment:
It looks like the `link.bridge` module is not used anywhere, it can be completely removed. Probably as separate commit.

Reply:
let's handle this after this PR will be merged.